workers/unity/Assets/Playground/Scripts/Cubes/ScoreCollision.cs

```diff
@@ -8,35 +8,32 @@ public class ScoreCollision : MonoBehaviour
     private EntityManager entityManager;
     private SpatialOSComponent component;
 
-    public void OnCollisionEnter(Collision col)
+    private void Start()
     {
-        if (entityManager == null)
+        component = GetComponent<SpatialOSComponent>();
+        if (!component)
         {
-            component = GetComponent<SpatialOSComponent>();
-            if (!component)
-            {
-                Debug.LogError("Could not get SpatialOSComponent.");
-                return;
-            }
-
-            entityManager = component.World.GetExistingManager<EntityManager>();
+            Debug.LogError("Could not get SpatialOSComponent.");
```

Comment:
Is this useful to log? What can the user do about it?

Reply:
I'll remove the entire check then

Reply:
I'm pretty sure it used to make sense, but is not relevant anymore. Feel free to remove it.